### PR TITLE
Fixes in the ability to run jupyter-lab: better startup script & build_python support

### PR DESCRIPTION
**Pull request by andy31415**
A wrong path for the paa store was hard to debug (would just say "attestation failed"). Changed the script to be verbose about the error and even try to adjust the path for the common use case of running from a sub-directory of a root checkout.

Also added ability for `build_python.sh` to install jupyter-lab requirements automatically.

### CODE REVIEW

File: src/controller/python/chip/ChipReplStartup.py [hunk 1]
```diff
@@ -77,50 +83,90 @@ def mattersetdebug(enableDebugMode: bool = True):
...
+    \t[red]caList[blue]:\t\t\t\tThe list of CertificateAuthority instances.
+    \t[red]caList[n][m][blue]:\t\t\tA specific FabricAdmin object at index m for the nth CertificateAuthority instance.''')
+
+    console.print(
+        f'\n\n[blue]Default CHIP Device Controller (NodeId: {devCtrl.nodeId}): '
+        f'has been initialized to manage [bold red]caList[0].adminList[0][blue] (FabricId = {caList[0].adminList[0].fabricId}), '
+        'and is available as [bold red]devCtrl')
+
+
+main()
```
Comment (tehampson) [inline src/controller/python/chip/ChipReplStartup.py:172]: Should this be:
```
if __name__ == "__main__":
    main()
```

**Review by tehampson — APPROVED** [review]

File: src/controller/python/chip/ChipReplStartup.py (hunk 1 above)
Comment (andy31415) [reply src/controller/python/chip/ChipReplStartup.py]: Apparently jupyter-lab does set name to main ... I did not expect that, but will make use of it. Changed!

File: scripts/build_python.sh [hunk 2]
```diff
@@ -132,6 +134,9 @@ while (($#)); do
             pregen_dir=$2
             shift
             ;;
+        --jupyter-lab | -j)
+            install_jupyterlab=yes
+            ;;
```
Comment (mkardous-silabs) [inline scripts/build_python.sh:139]: is there a shift missing here?

File: scripts/jupyterlab_requirements.txt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Requirements for running chip-repl in jypyterlab
```
Comment (mkardous-silabs) [inline scripts/jupyterlab_requirements.txt:1]:
```suggestion
# Requirements for running chip-repl in jupyterlab
```

**Review by mkardous-silabs — APPROVED** [review]
LGTM with a new nits

File: scripts/build_python.sh (hunk 2 above)
Comment (andy31415) [reply scripts/build_python.sh:139]: The others require a "yes" argument hence the shift. In this case I did not see the need to say `-j yes` or `-j no`. I believe the other arguments are due to historical usage so were never changed.

**Pull request merged**